.travis.yml

```diff
@@ -10,8 +10,11 @@ before_install:
 
 install:
     - travis_retry composer install
+    - (cd extra/intl-extension && travis_retry composer install)
 
-script: ./vendor/bin/simple-phpunit
+script:
+    - ./vendor/bin/simple-phpunit
+    - (cd extra/intl-extension && ./vendor/bin/simple-phpunit)
```

Comment:
what's about `./vendor/bin/simple-phpunit ./extra` in order to let simple_phpunit run future extensions in parallel?

Reply:
That does not work.